OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -151,6 +161,14 @@ std::vector<std::string> LiveFunctionsDataView::GetContextMenu(
   std::vector<std::string> menu;
   if (enable_select) menu.emplace_back(MENU_ACTION_SELECT);
   if (enable_unselect) menu.emplace_back(MENU_ACTION_UNSELECT);
+
+  // We have defined the behavior of these actions when only 1 function is
+  // selected.
```

Comment:
This is true, but probably won't help another developer do understand what's happening here.
Maybe something like: `// These actions only make sense when one function is selected, so we don't show them otherwise.`

Reply:
Yes, you are right. But for me makes sense, I imagine in a future a profiler selecting several functions, go to the first aparition and seeing in which order are executed with some "Go to next/prev".